resume.md

```diff
@@ -1,49 +1,43 @@
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+* **Languages**: Python, Java, JavaScript, Swift, SQL, R
+* **Libraries/Frameworks**: React, Pandas, NumPy, MongoDB, Next.js, jQuery, Flask, Scikit-Learn
+* **Version Control**: Git, GithubDesktop
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Product Engineer Intern**: Edith Labs | Cupertino, California | *September 2020 - Present*
+* Developed full-stack web app to connect students and mentors, in JavaScript, React, Next.js, HTML, CSS, and MongoDB.
+* Refactored data fetching using SWR hooks and spearheaded Ask a Question feature, boosting user engagement by 25%.
+* Created user stories and conducted A/B hypothesis tests to revamp mentor onboarding experience.
 
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
+**Data Science Intern**: T-Hub | Hyderabad, India | *June 2018 - July 2018*
+* Developed machine learning model for startup admission, decreasing work hours in review process by 75%; developed using Python, Pandas, and Scikit-Learn; model implemented by T-Hub admissions team. 
+* Processed and analyzed 7000-sample dataset with 42 features and consulted for 5 T-Hub startups on product and business development, improving retention, engagement, and profitability metrics by 15%-40%.
 
 ## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
+**Beacon Safety**: *iOS app to alert close contacts when unsafe* | tinyurl.com/beaconsafety | *May 2016 - Present*
+* Developed app using Swift, incorporating location tracking and SMS features.
+* Received 2300+ downloads and 5.0/5.0 average rating on iOS App Store, growing by 70% month-over-month.
+* Led 5 interns, performed user testing, and managed project development timelines.
```

Comment:
Vague - be more specific, there are a few themes here.  Make sure each of these items furthers your case why you're the best candidate as a software engineer